front/src/components/Edits/EditBlurb.tsx

```diff
@@ -1,14 +1,12 @@
 import React from 'react';
 import { Row, Col } from 'react-bootstrap';
-import {
-  WikiPageEditFragment,
-} from 'types/WikiPageEditFragment';
+import { StudyEditsHistoryQuery_study_wikiPage_edits } from 'types/StudyEditsHistoryQuery';
 import { Link } from 'react-router-dom';
 import ThemedButton from 'components/StyledComponents';
 
 interface EditBlurbProps {
-  edit: WikiPageEditFragment;
-  expanded: boolean;
+  edit: StudyEditsHistoryQuery_study_wikiPage_edits;
+  historyExpanded: any;
```

Comment:
Record<number, bool>, perhaps?

Reply:
Thanks @ericharding , just added types for this and `setExpanded`. Let me know if you notice anything else.